upstream/CVE-2021-25735.json

```diff
@@ -0,0 +1,89 @@
+{
+  "id": "CVE-2021-25735",
+  "modified": "2021-03-10T18:18:01Z",
+  "published": "2021-03-10T18:18:01Z",
+  "summary": "Validating Admission Webhook does not observe some previous fields",
+  "details": "A security issue was discovered in kube-apiserver that could allow node updates to bypass a Validating Admission Webhook. Clusters are only affected by this vulnerability if they run a Validating Admission Webhook for Nodes that denies admission based at least partially on the old state of the Node object. Validating Admission Webhook does not observe some previous fields.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:N/I:H/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.18.0"
+            },
+            {
+              "last_affected": "1.18.17"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:N/I:H/A:H"
+        }
+      ],
+      "ranges": [
```

Comment:
need to merge ranges in one array

Reply:
merged in https://github.com/aquasecurity/vuln-list-k8s/pull/85/commits/3b1a9be25c25d53eecbb0e6f831a795e1c379e15